utils.sh

```diff
@@ -30,8 +30,8 @@ function create_repo() {
 
 function set_policy() {
   local region=$1
-  RepositoryPermissionPolicy=$(printenv | grep RepositoryPermissionPolicy | sed 's/.*=//')
-  if [ -n "$RepositoryPermissionPolicy" ]
+  RepositoryPermissionPolicy=$(printenv RepositoryPermissionPolicy)
```

Comment:
If I just include the `if [ -n "${RepositoryPermissionPolicy}" ]` by itself, the script fails to load due to `RepositoryPermissionPolicy` being an unbound variable (and `set -u` being set). Perhaps there's another way around that beyond the realm of my shell scripting knowledge.